cmd/nitro/nitro.go

```diff
@@ -215,6 +215,9 @@ func mainImpl() int {
 				flag.Usage()
 				log.Crit("error opening parent chain wallet", "path", l1Wallet.Pathname, "account", l1Wallet.Account, "err", err)
 			}
+			if l1Wallet.OnlyCreateKey {
+				return 0
```

Comment:
Actually I think there's a couple other wallets we open below. We should add the same checks to them.

Reply:
Done.